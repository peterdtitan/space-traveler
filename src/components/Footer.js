import React from 'react';
import logo from '../assets/images/planet.png';

export default function Footer() {
  return (
    <footer className="flex flex-col text-white text-xs md:text-base md:flex-row items-center justify-center bg-slate-900 gap-10 fixed bottom-0 w-screen h-10 font-montserrat">
      <div className="flex gap-2 items-center">
        <img src={logo} alt="logo" className="w-8 invert" />
        <p className="text-xs md:text-base">Created by Microverse Inc under license</p>
      </div>
    </footer>
  );
}
