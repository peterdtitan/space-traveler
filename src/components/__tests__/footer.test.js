import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer component', () => {
  test('renders Footer component correctly', () => {
    const { container } = render(<Footer />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
