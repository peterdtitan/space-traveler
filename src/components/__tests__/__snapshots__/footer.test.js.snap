// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders Footer component correctly 1`] = `
<footer
  class="flex flex-col text-white text-xs md:text-base md:flex-row items-center justify-center bg-slate-900 gap-10 fixed bottom-0 w-screen h-10 font-montserrat"
>
  <div
    class="flex gap-2 items-center"
  >
    <img
      alt="logo"
      class="w-8 invert"
      src="planet.png"
    />
    <p
      class="text-xs md:text-base"
    >
      Created by Microverse Inc under license
    </p>
  </div>
</footer>
`;
